code/datums/ai_law_sets.dm

```diff
@@ -145,7 +145,6 @@
 	name = "Pranksimov"
 	law_header = "Comedy Routine"
 	selectable = TRUE
-	unique_ai = TRUE //honk
```

Comment:
[OOS] 

This var name offends me. `applicable_for_unique_ai_station_trait`, while verbose, would make WAY more sense for this as opposed to `unique_ai`.

Reply:
OOS meanin' outside of scope?

Reply:
Yeah